Azure/sonic-buildimage
# [sonic-host-service] Move to sonic-host-services package

**- Why I did it**

To move ‘sonic-host-service’ which is currently built as a separate package to ‘sonic-host-services' package. 

**- How I did it**

- Moved 'sonic-host-server' to 'src/sonic-host-services' and included it as part of the python3 wheel.
- Other files were moved to 'src/sonic-host-services-data' and included as part of the deb package.
- Changed build option ‘INCLUDE_HOST_SERVICE’ to ‘ENABLE_HOST_SERVICE_ON_START’ for enabling sonic-hostservice at boot-up by default.

**- How to verify it**

Ensure that 'sonic-host-server' is installed and runs based on the build option.

**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**


[sonic-host-service] Move to sonic-host-services package

**- A picture of a cute animal (not mandatory but encouraged)**
